draft-ietf-quic-http.md

```diff
@@ -182,6 +190,24 @@ itself as "hq-09-rickroll". Note that any label MUST conform to the "token"
 syntax defined in Section 3.2.6 of [RFC7230]. Experimenters are encouraged to
 coordinate their experiments on the quic@ietf.org mailing list.
 
+## Connection Reuse
+
+Once a connection exists to a server endpoint, this connection MAY be reused for
+requests with multiple different URI authority components.  The client MAY send
+any requests for which the client considers the server authoritative.
+
+An authoritative HTTP/QUIC endpoint is typically discovered because the client
+has received an Alt-Svc record from the request's origin which nominates the
+endpoint as a valid HTTP Alternative Service for that origin.  As required by
+{{RFC7838}}, clients MUST check that the nominated server can present a valid
+certificate for the origin before considering it authoritative. Clients SHOULD
+NOT assume that an HTTP/QUIC endpoint is authoritative for other origins without
```

Comment:
"SHOULD NOT assume"?  "cannot" or "MUST NOT" perhaps

Reply:
Yeah, I don't remember what my logic was for that one.  Probably trying to dance around the ORIGIN-style trust model.  Either is fine.